### DISCUSSION TITLE
[Question] Is there an easy way to transfer GitHub Issues to fider?

### DISCUSSION THREAD

**Discussion by Uatschitchun** [getfider/fider#939] [General]
This is simply a usage question. 
If I'm using Github's issue tracker to track feature requests right now and like to move them over to fider, what would be the easiest way to go?

**Comment by goenning** [comment]
Easiest option is to get issues using GH API and post using Fider API. If you create a script for that, please share, I'm sure more people would find it useful!

**Comment by Uatschitchun** [comment]
Thx! Thought that this would be the "easiest" Option, but hoped there was something existing already ;)
